src/devtools/useGotoAtomsSnapshot.ts

```diff
@@ -0,0 +1,16 @@
+import { useContext } from 'react'
+import { SECRET_INTERNAL_getStoreContext as getStoreContext } from 'jotai'
+import type { Scope } from '../core/types'
+
+import { isDevStore } from '../core/contexts'
+
+export function useGotoAtomsSnapshot(scope?: Scope) {
+  const StoreContext = getStoreContext(scope)
+  const store = useContext(StoreContext)
+
+  if (!isDevStore(store)) {
+    throw Error('useGotoAtomsSnapshot can only be used in dev mode.')
+  }
+  const restoreAtoms = store[3]
+  return restoreAtoms
```

Comment:
How about checking the misuse of `scope`?

```suggestion
  return useCallback((values: Parameters<typeof restore>[0]) => {
    for (const [atom] of values) {
      if (atom.scope !== scope) {
        throw new Error('atom scope mismatch to restore')
      }
    }
    restore(values)
  }, [restore]
```

Too much? (I renamed `restore` from `restoreAtoms`.)

Reply:
I should probably write some tests aswell?